README.md

```diff
@@ -148,7 +148,7 @@ JavaScript: SDK|Every Wednesday at 09:00 PT| [opentelemetry-calendar-js@googlegr
 PHP: SDK|Every Wednesday at 10:30 PT| [opentelemetry-calendar-php@googlegroups.com](https://groups.google.com/g/opentelemetry-calendar-php)                               | [Google Doc](https://docs.google.com/document/d/1WLDZGLY24rk5fRudjdQAcx_u81ZQWCF3zxiNT-sz7DI/edit)                        | [#otel-php](https://cloud-native.slack.com/archives/C01NFPCV44V)                                                                                             |
 Python: SDK|Every Thursday at 09:00 PT| [opentelemetry-calendar-python@googlegroups.com](https://groups.google.com/g/opentelemetry-calendar-python)                         | [Google Doc](https://docs.google.com/document/d/1CIMGoIOZ-c3-igzbd6_Pnxx1SjAkjwqoYSUWxPY8XIs/edit)                        | [#otel-python](https://cloud-native.slack.com/archives/C01PD4HUVBL)                                                                                          |
 Ruby: SDK|Every Tuesday at 09:00 PT| [opentelemetry-calendar-ruby@googlegroups.com](https://groups.google.com/g/opentelemetry-calendar-ruby)                             | [Google Doc](https://docs.google.com/document/d/1D15bO8o340sQm2CVZiukEJuCO_XMMHKPuTznoEhyFqE/edit)                        | [#otel-ruby](https://cloud-native.slack.com/archives/C01NWKKMKMY) and [GitHub Discussions](https://github.com/open-telemetry/opentelemetry-ruby/discussions) |
-Rust: SDK|N/A| [opentelemetry-calendar-rust@googlegroups.com](https://groups.google.com/g/opentelemetry-calendar-rust)                             | [Google Doc](https://docs.google.com/document/d/1tGKuCsSnyT2McDncVJrMgg74_z8V06riWZa0Sr79I_4)                             |                                                                                                 |
+Rust: SDK|Every Tuesday at 09:00 PT| [opentelemetry-calendar-rust@googlegroups.com](https://groups.google.com/g/opentelemetry-calendar-rust)                             | [Google Doc](https://docs.google.com/document/d/1tGKuCsSnyT2McDncVJrMgg74_z8V06riWZa0Sr79I_4)                             |                                                                                                 |
```

Comment:
Is it worth adding AM/PM to all the times?

Reply:
they are currently all in 24 hour time